envs/windows/base/Dockerfile

```diff
@@ -6,12 +6,12 @@ ARG BASE_IMAGE="mcr.microsoft.com/windows/servercore:1809"
 FROM ${BASE_IMAGE}
 SHELL ["cmd", "/S", "/C"]
 
-# see https://learn.microsoft.com/en-us/visualstudio/install/build-tools-container?view=vs-2019
-ADD https://aka.ms/vs/16/release/vs_buildtools.exe C:\temp\vs_buildtools.exe
+# see https://learn.microsoft.com/en-us/visualstudio/install/build-tools-container?view=vs-2022
+ADD https://aka.ms/vs/17/release/vs_buildtools.exe C:\temp\vs_buildtools.exe
 RUN C:\temp\vs_buildtools.exe --quiet --wait --norestart --nocache `
     --installPath C:\BuildTools `
     --add Microsoft.VisualStudio.Workload.VCTools --includeRecommended `
-    --add Microsoft.VisualStudio.Component.VC.14.27.x86.x64 `
+    --add Microsoft.VisualStudio.Component.VC.14.37.17.7.x86.x64 ^ `
```

Comment:
I see an additional `^`. Expected?

Reply:
Eagle eye! I've aligned it with what we have in our [other containers](https://devtopia.esri.com/Zurich-R-D-Center/containers/blob/main/windows/vs-buildtools/Dockerfile.base#L49).